src/components/AddContactForm.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { REGEX, MESSAGE_ERROS, TITLE } from '../constants/constants';
+import { validateInput } from '../helpers/helper';
+import './AddContactForm.css';
+
+class AddContactForm extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      firstName: '',
+      lastName: '',
+      imageUrl: '',
+      firstNameError: false,
+      lastNameError: false,
+      imageUrlError: false
+    };
+  }
+
+  onFirstNameInputChange = (event) => {
```

Comment:
Those 3 input change functions look pretty much the same. Here's some improvement you can bring here. Make only one generic function like:
`onFormInputChange = (event, type) => { ........... }` 'type' will be something like NAME or URL in your case.
If you add a name property on each input in jsx:
`<input name="firstName" />`
you can access that by using
`event.target.name`
so when you call the setState function it can be dynamically done like this:
`this.setState({ [event.target.name]: event.value })` so 'event.target.name' stays for 'firstName' in our example.
You can do the same with REGEX.NAME if you send the 'type' param as 'NAME'
Does it make sense? I can help if not!

Reply:
nice approach